template.yml.erb

```diff
@@ -671,6 +671,139 @@ Resources:
                           Value: !Ref BuildAndRunJava<%=name%>ProjectFunction
                 Period: 300
                 Stat: Sum
+
+  <%=name%>SlowCleanupTimeAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub "${SubDomainName}_<%=name.downcase%>_slow_cleanup_time"
+      AlarmDescription: p(95) cleanup time in Javabuilder's <%=name%> build and run lambda exceeded
+        the normal cleanup time for 15 minutes. Investigate if there has been a performance regression.
+      ActionsEnabled: true
+      OKActions: []
+      AlarmActions:
+        - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:javabuilder-slow-performance"]
```

Comment:
Is this topic defined in code somewhere?

Reply:
It is not (nor are any of our other topics). How should these be defined?